packages/admin-ui/src/date-picker/date-picker.state.ts

```diff
@@ -100,19 +99,16 @@ export const useDatePickerState = (props: DatePickerInitialState = {}) => {
   }, [autoFocus, dateFieldState.first])
 
   const invalid = useMemo(() => {
-    // const min = new Date(minDateValue)
-    // const max = new Date(maxDateValue)
-    // return value < min || value > max
+    const outOfBounds =
+      date < calendarState.minDateValue || date > calendarState.maxDateValue
 
-    return invalidProp
-  }, [invalidProp])
+    return invalidProp || outOfBounds
```

Comment:
Just a nitpick suggestion, it helped my comprehension of this const meaning. Feel free to accept or not
```suggestion
    return initialInvalid || outOfBounds
```

Reply:
also a suggestion: I probably would change the name to something like: `isInvalid` and `isOutOfBounds` to easily identify these variables as a boolean.

Reply:
Changed to tone